lib/hanami/cli.rb

```diff
@@ -1,7 +1,45 @@
-require "hanami/cli/version"
```

Comment:
@AlfonsoUceda I expect this API to change soon, it's a waste of time to write docs and unit tests for it. Integration tests are good to go in this phase.